### PR TITLE
docs: Update installation and use documentation

### PR DESCRIPTION
**Pull request by laruiss**
To make it clear which version of this component is for which version of Vue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +130,7 @@ Note the use of arrow functions, this is necessary because we are accessing
 
 ## Related
 
-- Focus Trap: https://github.com/davidtheclark/focus-trap
+- Focus Trap: <https://github.com/davidtheclark/focus-trap>
```
Comment (posva) [inline README.md:133]:
```suggestion
- Focus Trap: https://github.com/davidtheclark/focus-trap
```

File: README.md [hunk 2]
```diff
@@ -14,16 +22,28 @@ This library exports one single named export `FocusTrap` and **requires
...
+###  Register globally in a Vue 3 app
+
+```js
+import { FocusTrap } from 'focus-trap-vue'
+
+createApp(App)
+  .component('FocusTrap', FocusTrap)
+  .mount('#app)
+```
+
```
Comment (posva) [inline README.md:42]:
```suggestion

**Note this documentation is for Vue 3 and some props/events might not exist in the Vue 2 version**

```

**Review by posva — APPROVED** [review]
Thanks!

File: README.md (hunk 2 above)
Comment (posva) [inline README.md:42]:
```suggestion

**Note this documentation is for Vue 3 and some props/events might not exist in the Vue 2 version**

```

File: README.md (hunk 1 above)
Comment (posva) [inline README.md:133]:
```suggestion
- Focus Trap: https://github.com/davidtheclark/focus-trap
```

**Review by posva — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (laruiss) [reply README.md:133]: This was automatically corrected by Markdownlint ([MD034/no-bare-urls](https://github.com/DavidAnson/markdownlint/blob/v0.24.0/doc/Rules.md#md034)). I'll change it back to how it was.

**Pull request merged**